Predictor_submission_template.ipynb

```diff
@@ -182,11 +192,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "EXAMPLE_INPUT_FILE = \"validation/data/2020-08-01_2020-08-04_ip.csv\"\n",
+    "EXAMPLE_INPUT_FILE = \"validation/data/2020-09-30_historical_ip.csv\"\n",
```

Comment:
Had to re-generate the IP files. Only kept 2: one for historical data and one for a 'future' scenario.